geomesa-hbase/geomesa-hbase-dist/src/main/assemblies/component.xml

```diff
@@ -38,9 +38,10 @@
             <fileMode>0644</fileMode>
             <useProjectArtifact>false</useProjectArtifact>
             <useTransitiveDependencies>false</useTransitiveDependencies>
-            <useStrictFiltering>true</useStrictFiltering>
+            <!--<useStrictFiltering>true</useStrictFiltering>-->
```

Comment:
we should evaluate what the different jars are that get included with this change